libimage/manifest_list_test.go

```diff
@@ -38,3 +38,40 @@ func TestCreateManifestList(t *testing.T) {
 	require.Error(t, err)
 	require.Equal(t, errors.Cause(err), ErrNotAManifestList)
 }
+
+// Following test ensure that `Tag` tags the manifest list instead of resolved image.
+// Both the tags should point to same image id
+func TestCreateAndTagManifestList(t *testing.T) {
+
+	tagName := "testlisttagged"
+	listName := "testlist"
+	runtime, cleanup := testNewRuntime(t)
+	defer cleanup()
+	ctx := context.Background()
+
+	list, err := runtime.CreateManifestList(listName)
+	require.NoError(t, err)
+	require.NotNil(t, list)
+
+	manifestListOpts := &ManifestListAddOptions{All: true}
+	_, err = list.Add(ctx, "docker://fedora", manifestListOpts)
```

Comment:
Please use `busybox` instead of `fedora`.  It is much smaller and hence reduces the cost of CI.